src/ducks/connections/index.js

```diff
@@ -0,0 +1,42 @@
+import konnector from './konnector'
+
+// constant
+export const CREATE_CONNECTION = 'CREATE_CONNECTION'
+export const UPDATE_CONNECTION_RUNNING_STATUS = 'UPDATE_CONNECTION_RUNNING_STATUS'
+
+// reducers
+
+const reducer = (state = {}, action) => {
+  switch (action.type) {
+    case CREATE_CONNECTION:
+    case UPDATE_CONNECTION_RUNNING_STATUS:
+      const konnectors = { ...state }
+      konnectors[action.konnector.slug] = konnector(konnectors[action.konnector.slug], action)
+      return konnectors
```

Comment:
Same here:

```js
return { ...state, [action.konnector.slug: konnector(state[action.konnector.slug], action) }
```

If I remember well.

Reply:
Oooooh weeee I didn't know this one and it's so great !